python/lib/dependabot/python/file_parser.rb

```diff
@@ -93,12 +93,12 @@ def package_manager
       def detected_package_manager
         setup_python_environment if Dependabot::Experiments.enabled?(:enable_file_parser_python_local)
 
+        return PipenvPackageManager.new(T.must(detect_pipenv_version)) if detect_pipenv_version
+
         return PoetryPackageManager.new(T.must(detect_poetry_version)) if detect_poetry_version
```

Comment:
[seems the detection based on the package manager names in these files might be the reason?](https://github.com/dependabot/dependabot-core/blob/main/python/lib/dependabot/python/package_manager.rb#L53)

Reply:
for python , we are evaluating package managers based on the lock file names generated, i.e. poetry.lock , requirements.txt etc,